jekyll/_cci1/build-images-macos.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: classic-docs
+title: macOS Build Images
+categories: [build-images]
+description: macOS
+changefreq: "weekly"
+---
+
+CircleCI will try to run your build on macOS if we can detect an Xcode project.
+To change a project from building on Linux to building on macOS you can go to
+'Project Settings' -> 'Build Environment', and enable the 'Build OS X project'
+setting.
+
+**NOTE: To apply the new setting, you will need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding).**
+
+## Software
+
+We have multiple build images available, running OSX and macOS. The build image
+that your build is run on is dictated by the version of Xcode that you select
+in [your circle.yml file](/docs/1.0/ios-builds-on-os-x/#xcode-version).
```

Comment:
Thanks for doing this! Suggest to change the end of this sentence, to make it more clear why there is a link:
...the version of Xcode that you select in your `circle.yml` file, see the complete [list of Xcode version numbers](/docs/1.0/ios-builds-on-os-x/#xcode-version).
Otherwise looks good.
-Michelle